.github/workflows/release-python.yml

```diff
@@ -0,0 +1,90 @@
+name: Release Python
+
+on:
+  push:
+    branches:
+      - 135-python-bindings-distribution
+    tags:
+      - python/v*
+
+jobs:
+  macos:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        if: matrix.python-version != '3.7'
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Build wheels - x86_64
+        if: matrix.python-version == '3.7'
+        uses: messense/maturin-action@v1
+        with:
+          target: x86_64
+          args: -i python --release --manifest-path crates/python/Cargo.toml --out dist --sdist
+      - name: Build wheels - universal2
+        if: matrix.python-version != '3.7'
+        uses: messense/maturin-action@v1
+        with:
+          args: -i python --release --universal2 --manifest-path crates/python/Cargo.toml --out dist
+      - name: Install wheel
+        run: |
+          pip install qcs-sdk-python --no-index --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: dist
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        target: [x86_64, aarch64, armv7]
```

Comment:
Why armv7? Not saying we shouldn't include it, but (IIUC) it's not a particularly prevelant architecure.

Reply:
`armv7` runs on `armv8` as well, and you see it architecture on devices like the Raspberry PI. I believe some servers use it as well. I think we can just remove it until someone asks for it.